docs/03-tutorials/00-eventing/evnt-05-send-legacy-events.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Publish legacy events using Kyma Eventing
+---
+
+Kyma Eventing also supports sending and receiving of legacy events. In this tutorial we will show how to send legacy events using the Eventing Services.
+
+> **NOTE:** It is recommended to use [Cloud Events specification](https://cloudevents.io/) for sending and receiving events in Kyma.
+
+## Prerequisites
+
+1. Provision a [Kyma Cluster](../../02-get-started/01-quick-install.md).
+2. (Optional) Deploy [Kyma Dashboard](../../01-overview/main-areas/ui/ui-01-gui.md) on the Kyma cluster using the following command. Alternatively, you can also use `kubectl` CLI.
+   ```bash
+   kyma dashboard
+   ```
+
+## Create a Function
```

Comment:
In the other three tutorials, this section has the headline "Create a Workload". Why is it different here? The following line is identical for all four tutorials.

In general, I wonder whether the four nearly identical sets of instructions should really be maintained four times. If Busola changes their UI, you'll need to remember updating four instructions instead of just one. I recommend avoiding that.

Reply:
I updated it to `Create a Function`. Also, now removed it from some tutorials where possible and referring to the instructions given in the getting-started guide.

Reply:
That sounds great, thank you!